src/remodel_api/remodel.rs

```diff
@@ -28,6 +28,56 @@ fn xml_decode_options() -> rbx_xml::DecodeOptions {
     rbx_xml::DecodeOptions::new().property_behavior(rbx_xml::DecodePropertyBehavior::ReadUnknown)
 }
 
+fn get_required_string_option(options: &Table, option: &str) -> rlua::Result<String> {
+    let value = options.get(option).map_err(rlua::Error::external)?;
+
+    match value {
+        Value::String(value) => Ok(value.to_str().map_err(rlua::Error::external)?.to_string()),
+        Value::Nil => Err(rlua::Error::external(format!(
+            "The option {} must be specified",
+            option
+        ))),
+        _ => Err(rlua::Error::external(format!(
+            "The option {} must be a string",
+            option
+        ))),
+    }
+}
+
+fn get_string_option(options: &Table, option: &str, default: &str) -> rlua::Result<String> {
+    let value = options.get(option).map_err(rlua::Error::external)?;
+
+    match value {
+        Value::String(value) => Ok(value.to_str().map_err(rlua::Error::external)?.to_string()),
+        Value::Nil => Ok(default.to_string()),
+        _ => Err(rlua::Error::external(format!(
+            "The option {} must be a string",
+            option
+        ))),
+    }
+}
+
+fn get_bool_option(options: &Table, option: &str, default: bool) -> rlua::Result<bool> {
+    let value = options.get(option).map_err(rlua::Error::external)?;
+
+    match value {
+        Value::Boolean(value) => Ok(value),
+        Value::Nil => Ok(default),
+        _ => Err(rlua::Error::external(format!(
+            "The option {} must be a bool",
+            option
+        ))),
+    }
+}
+
+fn bool_into_query(boolean: bool) -> String {
+    if boolean {
+        String::from("True")
+    } else {
+        String::from("False")
+    }
+}
```

Comment:
Usage of this should not be pervasive enough to warrant a function; we should be able to keep the bool-to-query stuff entirely inside the code that actually does HTTP requests.